.github/workflows/linux-container-ci.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# $1 - Package Name
+# $2 - Builder version
+# $3 - Ecr Repo
+# $4 - Image Name
+
+set -e
+
+PACKAGE_NAME=$1
+shift
+BUILDER_VERSION=$1
+shift
+ECR_REPO=$1
```

Comment:
Does the repo ever change? Couldn't we just embed this? It's not secret really, it requires auth.

Reply:
Will update